### PR TITLE
BaSP-MR-147: Subscription react hook form implementation

### PR DESCRIPTION
**Pull request by marcohzo**
- Implement shared input
- Update all current forms and shared components used in the forms (inputs). Take into account styles for errors
- Adding validations to forms (required fields, dates, names, number of characters). Using the JOI library)
- Add reset function button to clear fields

### CODE REVIEW

**Comment by marcohzo** [comment]
[[Subscriptions] - React-hook-form implementation](https://trello.com/c/AadUNg8l/147-subscriptions-react-hook-form-implementation)

File: src/Components/Subscriptions/index.jsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ const Subscriptions = () => {
   const [titleModal, setTitleModal] = useState('');
   const [bodyModal, setBodyModal] = useState('');
   const [subscriptionId, setSubscriptionId] = useState('');
-
+  console.log(subscriptions);
```
Comment (fernandofernandezuy) [inline src/Components/Subscriptions/index.jsx]: remove this console.log()

**Comment by araefedg** [comment]
Checking that the solution pertaining to the ticket https://trello.com/c/h07cTV9V  was made in this PR, I give my OK, bug fixed, very good Marco!

**Review by Federico-Peralta — CHANGES_REQUESTED** [review]
You should pull the new changes from the master, merge your branch so you can have the changes because your form doesn't works now

**Review by Federico-Peralta — APPROVED** [review]

**Pull request merged**